src/map/utils/fishingutils.cpp

```diff
@@ -76,6 +84,28 @@ uint16 GetMessageOffset(uint16 ZoneID)
 	return MessageOffset[ZoneID];
 }
 
+// get player skill with item mod bonus added
+uint16 getPlayerFishingSkill(CCharEntity* PChar)
+{
+    uint16 skill = 0;
+
+    // player skill
+    skill += PChar->RealSkills.skill[SKILL_FISHING];
+
+#ifdef _DSP_SYNTH_DEBUG_MESSAGES_
```

Comment:
do we really need a define for this? I never liked that in the crafting utils either. Debug Messages are filterable and we could always comment/uncomment them. Its not even a synth.

Reply:
Done